utils/SwiftIntTypes.py

```diff
@@ -90,16 +90,16 @@ def numeric_type_names_Macintosh_only():
 
 def all_integer_binary_operator_names():
     return ['<<', '>>', '&*', '&', '&+', '&-', '|', '^']
-    
```

Comment:
What's the purpose of the blank spaces with no code after it?

From the current file in the repo:
https://github.com/apple/swift/blob/master/utils/SwiftIntTypes.py#L93

All that's there is space that's not needed.